index.ts

```diff
@@ -0,0 +1,28 @@
+import {install} from 'source-map-support';
```

Comment:
Didn't we already have the same issue in https://github.com/appium/node-teen_process/pull/253 ?

Reply:
...what?

Reply:
@jlipps could you please explain better what the actual issue was?

Reply:
as far as I could tell the problem there was with `tsconfig.json`, not `source-map-support`